doc/tutorial/specifics_quiz

```diff
@@ -0,0 +1,416 @@
+                                        </label>
+                                        <br/>
+                                    </li>
+                                    <li>
+                                        <input type="radio" value="D" />
+                                        <label for="D">
+                                            <pre>?       src/module/hello.py</pre>
+                                        </label>
+                                        <br/>
+                                    </li>
+                                </ol>
+                                <p class="answers B"><em>B</em></p>
+                            </div>
+                        </li>
+                        <li>
+                            <p class="question">How do you cherry pick from branch?</p>
+                            <div id="15" class="questions">
+                                <ol type="A">
+                                    <li><input type="radio" value="A" /><label for="A">fcm --cp branch_from branch_to</label><br/></li>
+                                    <li><input type="radio" value="B" /><label for="B">fcm --pick branch_from branch_to</label><br/></li>
+                                    <li><input type="radio" value="C" /><label for="C">fcm --cherry-pick branch_from branch_to</label><br/></li>
+                                    <li><input type="radio" value="D" /><label for="D">There is not proper support for this</label><br/></li>
+                                </ol>
+                                <p class="answers D"><em>D</em></p>
+                            </div>
+                        </li>
+                        <li>
+                            <p class="question">How can you discard some changes that have not been checked in yet?</p>
+                            <div id="16" class="questions">
+                                <ol type="A">
+                                    <li><input type="radio" value="A" /><label for="A">fcm rewind &lt;FILENAME1&gt; &lt;FILENAME2&gt; &lt;FILENAME3&gt;</label><br/></li>
+                                    <li><input type="radio" value="B" /><label for="B">fcm reverse &lt;FILENAME1&gt; &lt;FILENAME2&gt; &lt;FILENAME3&gt;</label><br/></li>
+                                    <li><input type="radio" value="C" /><label for="C">fcm discard &lt;FILENAME1&gt; &lt;FILENAME2&gt; &lt;FILENAME3&gt;</label><br/></li>
+                                    <li><input type="radio" value="D" /><label for="D">fcm revert &lt;FILENAME1&gt; &lt;FILENAME2&gt; &lt;FILENAME3&gt;</label><br/></li>
+                                </ol>
+                                <p class="answers D"><em>D</em></p>
+                            </div>
+                        </li>
+                        <li>
+                            <p class="question">How can you merge in the changes between two revisions?</p>
+                            <div id="17" class="questions">
```

Comment:
Might be better to use numbers rather than M and N here so as to remove any confusion as to which way round the two should be.